packages/storage-documents/src/on-demand-page-scan-batch-request.ts

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { StorageDocument } from '.';
+
+/**
+ * The client page scan run batch request document.
+ */
+export interface OnDemandPageScanBatchRequest extends StorageDocument {
+    scanRunBatchRequest: ScanRunBatchRequest[];
```

Comment:
why are we storing this as array in cosmos?

Reply:
to process by background function worker and transform into corresponding separate docs